nhs/data/handling.py

```diff
@@ -103,4 +106,68 @@ def read_spreadsheets(
             lambda x: x[0], placeholder_matches(files, file_dir_pattern, ["key"])
         )
 
-    return {key: val for key, val in zip(keys, map(reader, files))}
+    return {key: reader(file, sheet_id) for key, file in zip(keys, files)} if extension == "xlsx" else {
+        key: reader(file) for key, file in zip(keys, files)
+    }
+
+
+@log_entry_exit(level="INFO")
+def standarise_names(df_dict: dict[str, pl.LazyFrame | None], census_metadata: pl.LazyFrame | None,
+                     identification: str, abbreviation_column_name: str,
+                     long_column_name: str) -> dict[str, pl.LazyFrame | None]:
+    """
+    Process the column names of a polar Lazy frame dictionary to make them more readable.
+        1. Expand abbreviated names.
+        2. Converting to lower case.
+    If any parameters are None, this function returns prev_csv_pl_df without change.
+    The filename in both dictionaries are checked or filtered to be passed.
+
+    Parameters:
+    --------
+    param1 : dict[str, pl.LazyFrame]
+        df_dict: A dictionary containing census data as value and filename as key or None.
```

Comment:
Also not clear what should be `None`, this line implies the `key` should be `None` whereas it should be the lazy frames. Describe what the key should have (i.e. the format "G<int>" such as "G01"

Reply:
add in docstring df_dict: a dictionary of name-dataframe pair. Names must exist in the census_metadata[identification].